pkg/yurthub/cachemanager/cache_manager_test.go

```diff
@@ -2237,13 +2354,46 @@ func compareObjectsAndKeys(t *testing.T, objs []runtime.Object, namespaced bool,
 	return true
 }
 
+//used to reset the RESTMapper
+func resetRESTMapper(dStorage storage.Store, rm *hubmeta.RESTMapperManager) error {
```

Comment:
please add a `reset` func for `RESTMapperManager`

Reply:
The code has been updated, please review.